include/xrpl/protocol/detail/features.macro

```diff
@@ -29,6 +29,8 @@
 // If you add an amendment here, then do not forget to increment `numFeatures`
 // in include/xrpl/protocol/Feature.h.
 
+// fix1782: Check flags in Credential transactions
+XRPL_FIX    (1782,                       Supported::yes, VoteBehavior::DefaultNo)
```

Comment:
Please give this a meaningful name. We want people who don't have access to our internal ticketing system to still get some info from the name. `CredentialFlags` should be fine.

Reply:
renamed